presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -40,3 +40,5 @@ if [[ -z ${PRODUCT_TESTS_JAR} ]]; then
     PRODUCT_TESTS_JAR="${PRODUCT_TESTS_ROOT}/target/presto-product-tests-${PRESTO_VERSION}-executable.jar"
 fi
 export PRODUCT_TESTS_JAR=$(canonical_path ${PRODUCT_TESTS_JAR})
+DEFAULT_JDBC_DRIVER_JAR="${PROJECT_ROOT}/presto-jdbc/target/presto-jdbc-${PRESTO_VERSION}.jar"
```

Comment:
PRESTO_VERSION is undefined until we source presto.env. Please do it with an `if` statement as above - else it will blow up in presto-checks.